packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -781,15 +793,6 @@ contract SphinxUtils is SphinxConstants, StdUtils {
         address[] memory _owners,
         uint _threshold
     ) public pure returns (bytes memory safeInitializerData) {
-        require(
-            _owners.length > 0,
-            "Sphinx: You must have at least one owner in your 'sphinxConfig.owners' array."
-        );
-        require(
-            _threshold > 0,
-            "Sphinx: You must set your 'sphinxConfig.threshold' to a value greater than 0."
-        );
-
```

Comment:
I think we should keep these checks. Otherwise, this function will return initializer data that's bound to fail on-chain / yield an incorrect gnosis safe address. Kinda a footgun imo since this function is used in a few different contexts currently, and there may be more in the future